Rakefile

```diff
@@ -79,31 +79,37 @@ if saas?
       desc 'Run RSpec unit tests'
       RSpec::Core::RakeTask.new(:unit) do |task|
         task.pattern = 'spec/unit/**/*_spec.rb'
+        task.rspec_opts = '--require=spec_helper'
       end
 
       desc 'Run RSpec integration tests'
       RSpec::Core::RakeTask.new(:integration) do |task|
         task.pattern = 'spec/integration/**/*_spec.rb'
+        task.rspec_opts = '--require=spec_helper'
       end
 
       desc 'Run RSpec acceptance tests'
       RSpec::Core::RakeTask.new(:acceptance) do |task|
         task.pattern = 'spec/acceptance/**/*_spec.rb'
+        task.rspec_opts = '--require=spec_helper'
       end
 
       desc 'Run RSpec api tests'
       RSpec::Core::RakeTask.new(:api) do |task|
         task.pattern = 'spec/api/**/*_spec.rb'
+        task.rspec_opts = '--require=spec_helper'
       end
 
       desc 'Run RSpec use_cases tests'
       RSpec::Core::RakeTask.new(:use_cases) do |task|
         task.pattern = 'spec/use_cases/**/*_spec.rb'
+        task.rspec_opts = '--require=spec_helper'
       end
 
       desc 'Run RSpec test/s specified by input file pattern'
       RSpec::Core::RakeTask.new(:specific, :test_name) do | task, task_args |
         task.pattern = "#{task_args[:test_name]}"
+        task.rspec_opts = '--require=spec_helper'
```

Comment:
these changes should also be squashed in the commit introducing them - perhaps add a comment in the first line doing this to explain why it is needed.